.travis.yml

```diff
@@ -27,6 +23,13 @@ cache:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
 
+install: skip
+
 script:
   - ./.ci.sh
   - find . -type f -exec grep -l '^#!/bin/bash' {} + | xargs shellcheck
+
+env:
+  global:
+    # use `travis env set GH_TOKEN XXX` to set token to publish contents to GitHub pages
+    secure: XsGE9f9tLzIdqzKVCf6HyULznWPgw2k3Ghve+TAACsmSu1N1CpuNUdjfMS/9IdDncoP+AWV2aYZUAKImE/WyBvvXrgy8n3VpcCe13efU3fEXRkPQNBIZZOL1+/P7jeVTjX4cTe2ptemtJ2d4oFBcslJAX/c4D5HSu2/GMAfhe74=
```

Comment:
should this be in commit and public repo?

Reply:
**Short answer:** yes

**Longer answer:** This is a GitHub token encrypted with a public key. Travis has the private key and is the only one that can decrypt it. As the private key is bound to a repository (in this example ahus1/doctoolchain) this means: (a) only a build running on my repo can decrypt and use it and (b) it needs to be replaced with a encrypted token that works on the doctoolchain/doctoolchain repo. You can find more about it here: https://docs.travis-ci.com/user/encryption-keys/

**Correct answer:** This is only one way to pass environment variables to TravisCI. You can do this also via the Travis UI in the settings AFAIK (https://travis-ci.org/github/docToolchain/docToolchain/settings)